app/assets/javascripts/widgets/wysiwyg.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function(){
+  var anchorLinks = $('h3.anchorLinks a, a[href="#top"]');
+  var navHeight = $("#cu_nav").outerHeight();
+
+  /* offset omninav when interal anchor links scroll to target */
+  anchorLinks.on('click', function(e){
+    var target = $('[name=' + this.hash.slice(1) + ']');
```

Comment:
Is target an attribute of the `e` event object? I believe `e` usually has a `target` attribute. If it is one and the same, I would much prefer to use that.

What is the `this` value in this case? And what is `this.hash`?